packages/quantic/coveo.d.ts

```diff
@@ -12,6 +12,7 @@ interface Bindings {
 
 declare global {
   const CoveoHeadless: typeof HeadlessTypes;
+  const CoveoHeadlessCaseAssist: typeof HeadlessTypes;
```

Comment:
I think the CoveoHeadlessCaseAssist types are actually here: `./force-app/main/default/staticresources/coveoheadless/definitions/case-assist.index`. So a distinct import is likely needed.